packages/webview/src/component/objects/helper.ts

```diff
@@ -0,0 +1,25 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { KubernetesNamespacedObjectUI, KubernetesObjectUI } from './KubernetesObjectUI';
+
+export class ObjectHelper {
```

Comment:
I think the name is way too generic, I have no idea what it does (I mean, all helpers could be called ObjectHelper)

Reply:
I'll try to find a better one

Reply:
Changing to `KubernetesObjectUIHelper`, as the method it provides is working on `KubernetesObjectUI` objects